fdbserver/DataDistribution.actor.cpp

```diff
@@ -4641,12 +4641,18 @@ ACTOR Future<Void> storageServerFailureTracker(DDTeamCollection* self,
 						self->healthyZone.set(Optional<Key>());
 					}
 				}
+				if (!status->isUnhealthy()) {
+					// On server transistion from unhealthy -> healthy, trigger buildTeam check,
+					// handles scenario when team building failed due to insufficient healthy servers.
+					// Operaton cost is minimal if currentTeamCount == desiredTeamCount/maxTeamCount.
+					self->doBuildTeams = true;
+				}
```

Comment:
Doesn't this [line](https://sourcegraph.com/github.com/apple/foundationdb/-/blob/fdbserver/DataDistribution.actor.cpp?L4856) do the same thing? IMHO it's for the same purpose